docs/jwst/error_propagation/main.rst

```diff
@@ -0,0 +1,374 @@
+Description
+===========
+ 
+Steps in the pipelines calculate arrays for variances, or modify variance arrays 
+from <previous steps> in the pipeline.  In some cases these arrays are only used 
+internally to the current step.  For several steps, these arrays must be propagated to
+subsequent steps in the pipeline, or included in the cumulate error array.
+
+Stage 1 pipelines 
+-----------------
+Stage 1 pipelines perform detector-level corrections and ramp fitting for
+individual exposures, for nearly all imaging and spectroscopic modes. Details 
+of the pipelines can be found at `Stage 1 Pipelines
+<https://jwst-pipeline.readthedocs.io/en/latest/jwst/pipeline/calwebb_detector1.html>`_
+
+There are two unique configuration files to be used to control this pipeline, 
+depending on whether the data are to be treated as a Time Series Observation 
+(in which case some of the steps are set to be skipped by default).
+
+The steps to be executed differ for Near-IR and MIRI exposures. For 
+Near-IR, the steps are group_scale, dq_init, saturation, superbias, refpix, 
+linearity, persistence, dark_current, jump, ramp fitting, and gain scale.
+For MIRI, the steps are group_scale, dq_init, saturation, firstframe, lastframe,
+linearity, rcsd, dark_current, refpix, jump, ramp fitting, and gain scale.
+
+Of all of those steps, below are the particular steps that alter the ERR,
+VAR_POISSON, or VAR_RNOISE arrays of the science ramp data and output those
+altered arrays in the science product that is passed to the next step in the
+pipeline.  For the steps not listed below, those arrays do not exist or are
+propagated unchanged to the next pipeline step.
+
+
+Ramp_fitting
+++++++++++++
+This step calculates and populates the VAR_POISSON, VAR_RNOISE arrays
+(the variances and errors associated with ramp fitting) in the 'rate' and
+'rateints' files, and updates the ERR arrays in those files. The details of the 
+calculations can be found at `Ramp fitting 
+<https://jwst-pipeline.readthedocs.io/en/latest/jwst/ramp_fitting/main.html>`_
+
+
+
+Group_scale
```

Comment:
It's the ``gain_scale`` step that's applied after ``ramp_fit``, not ``group_scale``.